sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -185,7 +185,7 @@ describe("[AAD] TextAnalyticsClient", function() {
           assert.isAtLeast(aspect.confidenceScores.positive, 0);
           assert.isAtLeast(aspect.confidenceScores.negative, 0);
           assert.equal(aspect.offset, 32);
-          assert.equal(aspect.length, 6);
+          assert.equal(aspect.text.length, 6);
```

Comment:
Do you think it is worth checking this instead of removing the check itself?

Reply:
Yeah I think it is a good sanity check. Furthermore, it is easy to update it in the future if/when we add `length` back.